packages/core/src/users/usersStore.ts

```diff
@@ -100,9 +104,98 @@ const listenForLoadMoreAndFetch = ({state, instance}: StoreContext<UsersStoreSta
         group$.pipe(
           switchMap((e) => {
             if (!e.added) return EMPTY
-            const {batchSize, ...options} = parseUsersKey(group$.key)
+            const {userId, batchSize, ...options} = parseUsersKey(group$.key)
 
-            const projectId = options.projectId ?? instance.config.projectId
+            if (userId) {
+              // In the future we will be able to fetch a user from the global API using the resourceUserId,
+              // but for now we need to use the project subdomain to fetch a user
+              if (userId.startsWith('p')) {
```

Comment:
Should we make it clear in this comment that a userId starting with p is a project user?